client/src/java/com/zimbra/client/ZFolder.java

```diff
@@ -88,6 +89,7 @@
     private int mContentSequence;
     private int mImapUIDNEXT;
     private int mImapMODSEQ;
+    private String mAbsolutePath;
```

Comment:
@codacy-bot picked the wrong line in the file to comment on. I've removed the unused import